pkg/skaffold/deploy/kubectl.go

```diff
@@ -31,6 +36,35 @@ import (
 	"github.com/pkg/errors"
 )
 
+// Slightly modified from kubectl run --dry-run
+var deploymentTemplate = template.Must(template.New("deployment").Parse(`apiVersion: extensions/v1beta1
```

Comment:
Whats the tutorial for this example if you don't have the ports exposed through a service?
Are we going to make people kubectl expose them?

You need to hit the endpoint to get the log messages

Reply:
I was thinking port-forward.

Reply:
Isn't the EXPOSE directive suggesting that they actually want the port exposed to some degree? 

I'm not sure if users who are getting started are going to know that they have to run a specific `kubectl port-forward` to access their service

Reply:
It's a bit redundant I guess - you need to expose things at the pod level and then again later in a service/LB. I can create a service too - but creating a load balancer worries me a bit.

Basically anything that automatically opens up the cluster should be considered a little bit more carefully. Maybe we could prompt or something?

Reply:
Yeah I'm not sure, lets leave it as just a deployment now and then we'll try to get some feedback around if we should go ahead and template the service too.

Reply:
Yeah maybe we can automate this in something like https://github.com/GoogleCloudPlatform/skaffold/issues/94

The main reason I'm hesitant is that I worry something like this would happen:
https://blog.heptio.com/on-securing-the-kubernetes-dashboard-16b09b1b7aca?gi=7a65c1c530e2

If someone creates something with an LB and forgets about it later.

Reply:
Absolutely.  Maybe we can also solve this from our side with automatic port forwarding #94